### ISSUE TITLE
Breeding plan suggestions

### ISSUE THREAD

**Issue by joebywan** [cadon/ARKStatsExtractor#34]
Love the new breeding plan feature, I noticed it seems to just look at pairs and recommend based on if you win all of the stat rolls.

My suggestion would be to keep the current method looking at best possible breed based on chance, but also have a more conservative recommendation that looks at the whole breeding stock and makes recommendations based on being more likely to get consistent baby steps instead of swinging for the fences.

E.g. If I have:

- Male, max HP

- Female, max HP, max stamina

 - Female, max stamina, damage

The more consistent breed would be to mate the male with the female having max hp & stamina, then get a male from those two to mate with the female stamina/damage.

Whereas the current formula would recommend I keep breeding the male with the stamina/damage female giving a much larger chance of a poor breed.

Basically I'm asking for a roadmap to get all of the stats amalgamated with conservative breeds instead of going for the gamblers choice each time.

It's also likely to get more complex when breeding 2.0 comes out with the speculated inbreeding penalties.

**Comment by cadon** [comment]
Hi, good suggestion, I'll add it.

**Comment by cadon** [comment]
I included a new breeding-mode that should be better than the previous one (which is still there). If you feel it suggests not the right parents, give me a note. I think the algorithm could still make wrong suggestions in some strange cases.

**Comment by joebywan** [comment]
Awesome, will check it out.
